talos.nf

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
+
+include { VcfToMt } from './modules/talos/VcfToMt/main'
+include { ConvertPedToPhenopackets } from './modules/talos/ConvertPedToPhenopackets/main'
+include { MakePhenopackets } from './modules/talos/MakePhenopackets/main'
+include { GeneratePanelData } from './modules/talos/GeneratePanelData/main'
+include { QueryPanelapp } from './modules/talos/QueryPanelapp/main'
+include { FindGeneSymbolMap } from './modules/talos/FindGeneSymbolMap/main'
+include { RunHailFiltering } from './modules/talos/RunHailFiltering/main'
+include { ValidateMOI } from './modules/talos/ValidateMOI/main'
+include { HPOFlagging } from './modules/talos/HPOFlagging/main'
+include { CreateTalosHTML } from './modules/talos/CreateTalosHTML/main'
+
+
+process RunHailFilteringSV {
```

Comment:
Why is this process still part of the main workflow, rather than being off in a module? Also, is it actually called by anything?